src/utils/localized-block-data.ts

```diff
@@ -14,6 +14,14 @@ export function getBlockDataSourceType( blockName?: string ): string {
 	return getBlockConfig( blockName )?.dataSourceType ?? '';
 }
 
+export function getBlockTitle( blockName?: string ): string {
```

Comment:
As discussed, I have to tried to access the block's title from the global config so the prefix `remote-data-blocks` goes away.